openedx/core/djangoapps/programs/tasks/v1/tests/test_tasks.py

```diff
@@ -86,31 +78,23 @@ def _assert_num_requests(self, count):
     @mock.patch(UTILS_MODULE + '.get_completed_courses')
     def test_get_completed_programs(self, mock_get_completed_courses):
         """
-        Verify that completed programs are found, using the cache when possible.
```

Comment:
I think I got confused because I was no longer using the program config model which had a cache_ttl setting, but the catalog integration should have it as well. I will add it back in.